src/content/blog/2023-09-04-tus-node-server-v1.0.0.md

```diff
@@ -0,0 +1,172 @@
+---
+title: 'Tus Node.js 1.0.0: new packages, rewritten in TypeScript, and much more'
+author: murderlon
+date: 2023-09-04
+---
+
+The [Node.js tus server](https://github.com/tus/tus-node-server) has been
+revived and is now officially semver stable! It has been completely rewritten in
+TypeScript, split into separate packages, has new events and hooks system,
+elaborate docs with examples, and many fixes and features we’ll dive into below.
+
+## History
+
+The Node.js tus server, [tus-node-server][], was originally published by
+[@bhstahl][] in May 2016. As with many open source projects, not everyone has
+the time to maintain things for free in their own time and as such
+[tus-node-server][] was more or less abandoned at the end of 2018.
+
+In 2021, [@acconut][], [@mitjab][], and others sprung some live into the server
+again by upgrading dependencies adding some missing essential features. However
+there was still a lot of work ahead before the server could be considered stable
+and on par with the [protocol specification][].
+
+At the start of 2023, [Transloadit][] graciously allowed me ([@murderlon][]) to
+work on the server while getting paid. This, together with the help of
```

Comment:
Maybe we could link to https://transloadit.com/open-source/, where more details about Transloadit's open-source commitment are available.

Reply:
>https://transloadit.com/open-source/

As folks with skin in the game, maybe also do a quick review on that page to see if it still makes sense and is written in the right tone of voice